edm4hep.yaml

```
@@ -270,290 +270,290 @@ datatypes:
+      - float                iTheta          // intrinsic direction of cluster at position  Theta. Not to be confused with direction cluster is seen from IP.
+      - float                phi             // intrinsic direction of cluster at position - Phi. Not to be confused with direction cluster is seen from IP.
+      - edm4hep::Vector3f    directionError  // covariance matrix of the direction (3 Parameters) [mm^2].
     VectorMembers:
-      - float   shapeParameters      //shape parameters - check/set collection parameter ClusterShapeParameters for size and names of parameters.
-      - float   subdetectorEnergies  //energy observed in a particular subdetector. Check/set collection parameter ClusterSubdetectorNames for decoding the indices of the array.
+      - float   shapeParameters      // shape parameters. This should be accompanied by a descriptive list of names in the shapeParameterNames collection level metadata, as a vector of strings with the same ordering.
+      - float   subdetectorEnergies  // energy observed in a particular subdetector. Check/set collection parameter ClusterSubdetectorNames for decoding the indices of the array.
     OneToManyRelations:
-      - edm4hep::Cluster        clusters     //clusters that have been combined to this cluster.
-      - edm4hep::CalorimeterHit hits         //hits that have been combined to this cluster.
-      - edm4hep::ParticleID     particleIDs  //particle IDs (sorted by their likelihood)
+      - edm4hep::Cluster        clusters     // clusters that have been combined to this cluster.
+      - edm4hep::CalorimeterHit hits         // hits that have been combined to this cluster.
+      - edm4hep::ParticleID     particleIDs  // particle IDs (sorted by their likelihood)
 
   #-------------  TrackerHit
   edm4hep::TrackerHit:
     Description: "Tracker hit"
-    Author: "F.Gaede, DESY"
+    Author: "EDM4hep authors"
     Members:
-      - uint64_t cellID      //ID of the sensor that created this hit
-      - int32_t type                       //type of raw data hit, either one of edm4hep::RawTimeSeries, edm4hep::SIMTRACKERHIT - see collection parameters "TrackerHitTypeNames" and "TrackerHitTypeValues".
-      - int32_t quality                    //quality bit flag of the hit.
-      - float time                     //time of the hit [ns].
-      - float eDep                     //energy deposited on the hit [GeV].
-      - float eDepError                //error measured on EDep [GeV].
-      - edm4hep::Vector3d position     //hit position in [mm].
-      - std::array<float,6> covMatrix  //covariance of the position (x,y,z), stored as lower triangle matrix. i.e. cov(x,x) , cov(y,x) , cov(y,y) , cov(z,x) , cov(z,y) , cov(z,z)
+      - uint64_t cellID      // ID of the sensor that created this hit.
+      - int32_t type                       // type of raw data hit, either one of edm4hep::RawTimeSeries, edm4hep::SIMTRACKERHIT.
```

```suggestion
      - int32_t type                       // type of raw data hit.
```
We don't define anything like that yet, so I think it's best to just remove it completely.